docs/src/components/productX/XChartsDemo.tsx

```diff
@@ -1,109 +1,86 @@
+  sx={{
+    '& .MuiMarkElement-root': { display: 'none' },
+    '& .MuiMarkElement-root.MuiMarkElement-highlighted': { display: 'block' },
+  }}
+/>`;
 
 export default function XChartsDemo() {
   return (
     <Frame sx={{ height: '100%' }}>
       <Frame.Demo sx={{ p: 2, flexGrow: 1 }}>
         <Paper
           variant="outlined"
-          sx={[
-            {
-              '& > div': {
-                borderRadius: 1,
-                bgcolor: '#fff',
-              },
-            },
-            (theme) =>
-              theme.applyDarkStyles({
-                '& > div': {
-                  bgcolor: 'primaryDark.900',
-                },
-              }),
-          ]}
+          sx={(theme) => ({
+            maxWidth: '100%',
+            mx: 'auto',
+            bgcolor: '#FFF',
+            height: '100%',
+            borderRadius: '8px',
+            ...theme.applyDarkStyles({
+              bgcolor: 'primaryDark.900',
+            }),
+          })}
         >
-          <Box sx={{ py: 1 }}>
-            <LineChart
+          <Box sx={{ p: 2 }}>
+            {/* <LineChart
               sx={{
                 '& .MuiMarkElement-root': { display: 'none' },
                 '& .MuiMarkElement-root.MuiMarkElement-highlighted': { display: 'block' },
               }}
               margin={{ top: 10, bottom: 50, left: 40, right: 20 }}
               series={[
                 { data: [35, 44, 24, 34], stack: 'total', area: true },
-                { data: [51, 6, 49, 30], stack: 'total', area: true },
-                { data: [5, 15, 30, 50], stack: 'total', area: true },
+                { data: [51, 32, 49, 30], stack: 'total', area: true },
+                { data: [5, 31, 30, 50], stack: 'total', area: true },
                 { data: [60, 50, 15, 10], stack: 'total', area: true },
               ]}
               xAxis={[{ data: ['Q1', 'Q2', 'Q3', 'Q4'], scaleType: 'point' }]}
               colors={mangoFusionPalette}
               height={250}
-            />
+            /> */}
             <BarChart
-              sx={{
-                '& .MuiMarkElement-root': { display: 'none' },
-                '& .MuiMarkElement-root.MuiMarkElement-highlighted': { display: 'block' },
-              }}
-              margin={{ top: 10, bottom: 30, left: 40, right: 10 }}
               series={[
                 { data: [35, 44, 24, 34] },
                 { data: [51, 6, 49, 30] },
                 { data: [15, 25, 30, 50] },
                 { data: [60, 50, 15, 25] },
               ]}
+              height={250}
               xAxis={[{ data: ['Q1', 'Q2', 'Q3', 'Q4'], scaleType: 'band' }]}
-              height={200}
+              margin={{ top: 10, bottom: 30, left: 40, right: 10 }}
               colors={mangoFusionPalette}
+              sx={{
+                '& .MuiMarkElement-root': { display: 'none' },
+                '& .MuiMarkElement-root.MuiMarkElement-highlighted': { display: 'block' },
```

Comment:
It overflows

```suggestion
                '& .MuiMarkElement-root.MuiMarkElement-highlighted': {
                  display: 'block',
                },
```

![Screenshot 2023-11-25 at 13 12 35](https://github.com/mui/material-ui/assets/3165635/2ff89407-acf8-4826-bfc9-e8afd6c8ffe4)

Reply:
Actually, I think we can remove the whole thing, `MuiMarkElement` are no longer rendered, it was for this:

<img src="https://github.com/mui/material-ui/assets/3165635/bb923a57-a1d1-45f4-97ab-c23b5c4ed5c4" width="464">